tachyon_core/src/ffi.rs

```diff
@@ -113,20 +114,43 @@ pub unsafe extern "C" fn tachyon_stream_delete(connection: *mut Connection, stre
 pub unsafe extern "C" fn tachyon_stream_check_exists(
     connection: *const Connection,
     stream: *const c_char,
-) -> bool {
+    out: *mut *mut c_void,
```

Comment:
We need doc comments for SAFETY and the return value if we're adding an `out` parameter. You can check the other functions and just copy-paste from those doc comments